typing/typedecl.ml

```diff
@@ -95,7 +95,7 @@ let add_type ~check id decl env =
   Builtin_attributes.warning_scope ~ppwarning:false decl.type_attributes
     (fun () -> Env.add_type ~check id decl env)
 
-let enter_type rec_flag env sdecl (id, uid) =
+let enter_type ~abstract_abbrevs rec_flag env sdecl (id, uid) =
```

Comment:
the documentation comment in trunk could be also reused here (but it needs a small tweak as `abstract_abbrevs` is a boolean here rather than an option)